WORKSPACE

```diff
@@ -12,30 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-http_archive(
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
+
+git_repository(
     name = "build_bazel_rules_apple",
-    url = "https://github.com/bazelbuild/rules_apple/archive/0.3.0.zip",
-    strip_prefix = "rules_apple-0.3.0",
-    # Generated by running: openssl sha -sha256 <path to zip>
-    sha256 = "0c2f7b903c2334a3e7e0b565f0f80a854047ac36202978318f853cb762bf33f3"
+    remote = "https://github.com/bazelbuild/rules_apple.git",
+    tag = "0.9.0",
+)
+
+load(
+    "@build_bazel_rules_apple//apple:repositories.bzl",
+    "apple_rules_dependencies",
 )
 
+apple_rules_dependencies()
+
 git_repository(
-    name = "bazel_skylib",
-    remote = "https://github.com/bazelbuild/bazel-skylib.git",
-    tag = "0.2.0",
+    name = "build_bazel_rules_swift",
```

Comment:
Tip: Was `bazel_skylib` renamed to `build_bazel_rules_swift`?

Reply:
I don't believe so. I'm not sure why bazel_skylib was here before but it doesn't appear to be required anymore.